### ISSUE TITLE
term_cols & term_rows ignored

### ISSUE THREAD

**Issue by noscript** [vim/vim#6861]
**Describe the bug**
`term_cols` and `term_rows` options to `term_start()` don't affect the terminal size, instead `termwinsize` takes the precedence.

**To Reproduce**
```sh
vim -g -u NONE +'set termwinsize=20x80' +'call term_start("stty size", #{term_rows: 50, term_cols: 100})'
```
**Expected behavior**
`50 100`

**Actual behavior**
`20 80`

**Environment:**
- Vim 8.2.1578
- OS: Ubuntu 18.04
- Terminal: GUI

**Comment by lacygoill** [comment]
Fixed by [8.2.1589](https://github.com/vim/vim/releases/tag/v8.2.1589): *term_start() options for size are overruled by 'termwinsize'*

**Comment by k-takata** [comment]
Closing, then.
